documentation/modules/managing/con-cluster-loss-recovery.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-loss_{context}"]
+= Cluster loss
```

Comment:
Again, I'm not sure this title is descriptive enough.

Reply:
Changed to _Recovery from loss of a Kubernetes cluster_